Sources/Plasma/Apps/plClient/CMakeLists.txt

```diff
@@ -7,18 +7,59 @@ if(MOVIE_AVAILABLE)
     include_directories(${Opus_INCLUDE_DIR})
 endif()
 
-# Test for Python Interpreter, which will be used for extra build scripts if available
-if(Python3_Interpreter_FOUND)
-    include(FindPythonModule)
+set(external_SCRIPTS
+    external/create_resource_dat.py
+    external/makeres.py
+    external/render_svg.py
+    external/scalergba.py
+)
 
-    # Test for Python modules needed to build resource.dat
-    find_python_module(cairosvg)
-    find_python_module(PIL)
+set(external_SOURCES
+    external/Cursor_Base.svg
+    external/Linking_Book.svg
+    external/Loading_Text_rasterfont.svg
+    external/Voice_Chat.svg
+)
 
-    if((DEFINED PY_CAIROSVG) AND (DEFINED PY_PIL))
-        set(CAN_BUILD_RESOURCE_DAT ON)
-    endif((DEFINED PY_CAIROSVG) AND (DEFINED PY_PIL))
-endif(Python3_Interpreter_FOUND)
+find_program(
+    PNGCRUSH_EXECUTABLE pngcrush
+    DOC "Path to pngcrush"
+)
+
+# Can we generate the resource.dat file?
+python_install_modules(REQUIREMENTS_FILE "${CMAKE_CURRENT_SOURCE_DIR}/external/requirements.txt")
```

Comment:
If we're loading requirements here now, we should probably just remove [requirements.txt](https://github.com/H-uru/Plasma/blob/master/requirements.txt)

Reply:
This is potentially a no-op. Right now, it only functions if the user `-DENABLE_PIP_INSTALL=ON`. Later, if we detect that we are using the vcpkg submodule,. then we'll actually download the requirements here.

Reply:
Good point.  I suppose it retains its utility for now as a central requirement list.